cdb2api/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(src
 
 list(APPEND src ${PROJECT_SOURCE_DIR}/util/walkback.c)
 set_source_files_properties(${PROJECT_SOURCE_DIR}/util/walkback.c PROPERTIES COMPILE_FLAGS -DUSE_UNWIND)
+set_source_files_properties(cdb2api.c PROPERTIES COMPILE_FLAGS "-Wno-stringop-overflow")
```

Comment:
https://gcc.gnu.org/onlinedocs/gcc/Diagnostic-Pragmas.html
Lets do it for just the offending section - not entire file.